src/components/smart_objects/test/CMakeLists.txt

```diff
@@ -53,5 +53,11 @@ set(EXCLUDE_PATHS
   SmartObjectConvertionTime_test.cc
 )
 
+# Enable detect Double-free, invalid free
+# AddressSanitizer is a fast memory error detector. 
+# It consists of a compiler instrumentation module and a run-time library.
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fno-omit-frame-pointer -fsanitize=address")
```

Comment:
@SNiukalov do you really need address sanity enabled for unit tests? I believe this tool for debug purposes only

Reply:
@AKalinich-Luxoft 
Yes you are right. But without this option, we can't detect Double-free, becuase test always will finish normal. But with this option test will be aborted and show backtrace.